code/modules/exosuits/interface/_interface.dm

```diff
@@ -0,0 +1,216 @@
+#define BAR_CAP 12
```

Comment:
You should `#undef` this at the end of the file, or else it belongs in __defines